code/__DEFINES/robots.dm

```diff
@@ -36,6 +36,7 @@
 #define MED_BOT				(1<<4)	//!  Medibots
 #define HONK_BOT			(1<<5)	//!  Honkbots & ED-Honks
 #define FIRE_BOT			(1<<6)  //!  Firebots
+#define HUG_BOT 			(1<<7)  //!  Hugbots //NSV13
```

Comment:
I would suggest this goes in a modular file, if this is not possible I'm OK with taking this as is.